fetcher/jvn/jvn.go

```diff
@@ -65,26 +66,21 @@ func FetchJVN() (models.FetchedCPEs, error) {
 	return fetched, nil
 }
 
-func makeJvnURLs(years []int) (urls []string) {
-	latestFeeds := []string{
-		"https://jvndb.jvn.jp/ja/rss/jvndb_new.rdf",
-		"https://jvndb.jvn.jp/ja/rss/jvndb.rdf",
-	}
-
-	if len(years) == 0 {
-		return latestFeeds
-	}
-
+func makeJvnURLs() (urls []string, err error) {
```

Comment:
Names needed?

Reply:
It is possible to implement it without it, but since it has been written using named return values ​​from before, I have left it as it is.

Reply:
Got it!

Reply:
https://github.com/vulsio/go-cpe-dictionary/compare/c789463945627f90561b947ffe5c1f50f4670fa4..5970929c1f80d9bf880683e5037997d95f3ff94d